releasenotes/README.md

```diff
@@ -12,11 +12,28 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the previous release notes.](./v50.md)
 
-## Sep 2, 2021
+## Sept 9, 2021
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
- * FIX: When running `force:source:deploy -m`, you can now include a space when specifying the metadata type, as long as you enclose the parameter value in double quotes. Previously, including a space, such as `-m "ApexClass: MyClass"` returned the error `No source backed components present in the package.` ([GitHub issue #1151](https://github.com/forcedotcom/cli/issues/1151))
+* FIX: The `force:source:deploy|retrieve|convert` commands correctly support these metadata types:
+   
+    * AssignmentRule
+    * AuraDefinitionBundle
+    * AutoResponseRule
+    * RestrictionRule 
+    * SharingCriteriaRule
+    * SharingOwnerRule
+
+    ([GitHub issue #1158](https://github.com/forcedotcom/cli/issues/1158))
+    
+* FIX: Because the Metadata API doesn't support the CustomFieldTranslation metadata type, the `force:source:deploy|retrieve|convert` commands now correctly exclude it from the `package.xml` file. Previously it wasn't excluded, and when you ran the commands you'd get the error `Unknown type name 'CustomFieldTranslation' specified in package.xml`. 
+
+* FIX: The `force:source:deploy|retrieve|convert` commands now support the AuraDefinitionBundle metadata type being located within a CustomObject folder in your project. Previously the commands would return an error similar to this one: `ERROR running force:source:retrieve: Cannot read property 'id' of undefined`. ([GitHub issue #1148](https://github.com/forcedotcom/cli/issues/1148))
```

Comment:
This isn't worded quite right.  The fix for this situation (an aura file within a CustomObject folder) now throws an error stating exactly what the problem is.  Something like, "invalid child [path/to/file] of parent [parent type] found.

Reply:
Updated.